frontend/components/RangeSlider.jsx

```diff
@@ -23,15 +27,34 @@ export default class RangeSlider extends Component {
     const { value } = this.state;
 
     this.props.onChange({ target: { name, value: String(value) } });
+    this.setState({
+      value: Math.floor((this.props.min + this.props.max) / 2),
+      userHasInteracted: false,
+    });
   }
 
   render() {
     const {
       props: { min, max },
       state: { value },
     } = this;
+
+    let rangeValue = '';
+
```

Comment:
It's splitting hairs given the result is the same but defaulting JSX to `null` seems more prevalent than `''`